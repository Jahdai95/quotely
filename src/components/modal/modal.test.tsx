import { render } from "@testing-library/react";
import { describe, vi, it, expect } from "vitest";
import Modal from "./modal";
import { QuotesProvider } from "../../providers/quotes.provider";

describe("Should render Modal component", () => {
  const getRender = () => {
    return render(
      <QuotesProvider>
        <Modal handleClose={() => vi.fn()} onSave={() => vi.fn()} open>
          <h2>Contenido del modal</h2>
        </Modal>
      </QuotesProvider>
    );
  };

  it("Should macth snapshot", () => {
    const { container } = getRender();
    expect(container).toMatchSnapshot();
  });
});
