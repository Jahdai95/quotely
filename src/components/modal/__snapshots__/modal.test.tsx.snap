// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render Modal component > Should macth snapshot 1`] = `
<div
  aria-hidden="true"
/>
`;
